documentation/library_guide/parallel_api/additional_algorithms.rst

```diff
@@ -76,17 +76,26 @@ header.  All algorithms are implemented in the ``oneapi::dpl`` namespace.
     search sequence: [0, 2, 4, 7, 6]
     result sequence: [1, 4, 8, 10, 10]
 
-* ``sort_by_key``: performs a stable key-value sort. The algorithm sorts the sequence's keys according to 
-  a comparioson operator. If no comparator is provided, then the elements are compared with ``operator<``.
-  The sequence's values are permutated according to the sorted sequence's keys. The prerequisite for correct
-  behavior is that the size for both keys sequence and values sequence shall be the same.  
+* ``sort_by_key``: performs a key-value sort. The algorithm sorts a sequence of keys according to
+  a comparison operator. If no comparator is provided, then the elements are compared with ``operator<``.

```

Comment:
Probably, should be `std::less` ? 
oneDPL impl uses its own predicate with "operator<", right... But, probably, should we change it as well?

Reply:
`std::sort` uses `operator<` until c++20, and `std::less` since c++20.
I suppose it does not result in backward compatibility issues since `std::less` is defined as:

> Function object for performing comparisons. The main template invokes operator< on type T.

Namely, `std::less` is a superset including `operator<`.

However, it is a change in the function signature, and it may break the ABI for users. Please, correct me if I am mistaken here.

Reply:
Ah, just ignore the note about ABI. There is no change in the signature.

Reply:
There are plenty of algorithms, which should be changed accordingly. Would you mind if it is done separately?

Reply:
I don't  mind. Lets create an issue to don't forget.

Reply:
Done: https://github.com/oneapi-src/oneDPL/issues/1779